common/djangoapps/student/views.py

```diff
@@ -1869,6 +1885,30 @@ def _record_registration_attribution(request, user):
         UserAttribute.set_user_attribute(user, REGISTRATION_AFFILIATE_ID, affiliate_id)
 
 
+def _record_utm_registration_attribution(request, user):
+    """
+    Attribute this user's registration to the latest UTM referrer, if
+    applicable.
+    """
+    utm_cookie_name = RegistrationCookieConfiguration.current().utm_cookie_name
+    utm_cookie = request.COOKIES.get(utm_cookie_name)
+    if user is not None and utm_cookie is not None:
+        utm = json.loads(utm_cookie)
+        for utm_parameter in REGISTRATION_UTM_PARAMETERS:
+            UserAttribute.set_user_attribute(
+                user,
+                REGISTRATION_UTM_PARAMETERS.get(utm_parameter),
+                utm.get(utm_parameter)
+            )
+        created_at_unixtime = int(utm.get('created_at', 0))
```

Comment:
Shouldn't cause downstream issues, but let's doublecheck.

@mulby perhaps you can weigh in on this?  prefer `1970-1-1` if this is not set for some reason?  Or `null`